### DISCUSSION TITLE
Discussion of defaults on inspect_reduced_spectrum()

### DISCUSSION THREAD

**Discussion by cylammarco** [cylammarco/ASPIRED#45] [General]
*Moved from Issue #9 to here*

### @rjs3273 commented on Mar 19, 2020

What is the proper venue for discussions? The fact this page is called "issues" sort of implies something is wrong. I would not go that far. I just want to raise a question about selecting parameter defaults.

Currently we have

inspect_reduced_spectrum(self, wave_min=4000., wave_max=8000., ...

That is awfully SPRAT specific. Whilst that is obviously best for me and makes my life easier playing with SPRAT data, it seems to me that better defaults would either a) the entire detector array or b) some clever automated guess based on, for example where the ap_trace fails and drops to zero. I would suggest that a is enough for release v1 and you can add b to the ideas list.

### @cylammarco commented on May 7, 2020
For long-slit spectrograph, the uncontaminated spectral range is about 4000A. So, I think the default blue-end can be as short as 3000A, and the default red-end can be as long as 9000A:

Below 3000, it would be space-based UV observation. Those missions probably have enough budget to build their own pipelines.

Above 9000, it would be mostly dominated by sky noise. We are not saying this is designed for only optical long-slit spectral data, but that is the most common kind of low-cost spectrograph at low-budget facilities.

So, I think 3500 - 8500A with a range_tolerance of 500A should be a sensible choice for a low-resolution spectrograph that is aimed for maximum uncontaminated wavelength coverage.

**Comment by rjs3273** [comment]
I see the defaults are now 3500 - 8500A. Looks good to me. My plots are coming out nicely.
